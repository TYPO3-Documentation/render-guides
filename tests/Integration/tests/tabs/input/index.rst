==============
Tabs
==============

..  tabs::

    ..  group-tab:: bash

        ..  code-block:: bash

            composer create-project typo3/cms-base-distribution:^11 example-project-directory

    ..  group-tab:: powershell

        ..  code-block:: powershell

            composer create-project "typo3/cms-base-distribution:^11" example-project-directory

..  tabs::

    ..  group-tab:: bash

        ..  code-block:: bash

            touch example-project-directory/public/FIRST_INSTALL

    ..  group-tab:: powershell

        ..  code-block:: powershell

            echo $null >> public/FIRST_INSTALL
