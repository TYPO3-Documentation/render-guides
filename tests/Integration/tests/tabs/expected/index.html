<!-- content start -->
    <div class="section" id="tabs">
    <h1>Tabs</h1>

            <div class="sphinx-tabs docutils du-container">
    <div aria-label="Tabbed content" class="closeable" role="tablist">
        <button aria-controls="panel-ash" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-ash" name="ash" role="tab" tabindex="0">bash</button>
<button aria-controls="panel-owershell" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-owershell" name="owershell" role="tab" tabindex="0">powershell</button>
    </div>
    <div aria-labelledby="tab-ash" class="sphinx-tabs-panel group-tab" name="ash" id="panel-ash" role="tabpanel" tabindex="0">
    <div class="highlight-bash notranslate">
        <div class="highlight">
            <pre><code class=""><span>composer create-project typo3/cms-base-distribution:^11 example-project-directory</span></code></pre>
        </div>
    </div>
</div>
<div aria-labelledby="tab-owershell" class="sphinx-tabs-panel group-tab" name="owershell" id="panel-owershell" role="tabpanel" tabindex="0">
    <div class="highlight-powershell notranslate">
        <div class="highlight">
            <pre><code class=""><span>composer create-project &quot;typo3/cms-base-distribution:^11&quot; example-project-directory</span></code></pre>
        </div>
    </div>
</div>
</div>

            <div class="sphinx-tabs docutils du-container">
    <div aria-label="Tabbed content" class="closeable" role="tablist">
        <button aria-controls="panel-ash" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-ash" name="ash" role="tab" tabindex="0">bash</button>
<button aria-controls="panel-owershell" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-owershell" name="owershell" role="tab" tabindex="0">powershell</button>
    </div>
    <div aria-labelledby="tab-ash" class="sphinx-tabs-panel group-tab" name="ash" id="panel-ash" role="tabpanel" tabindex="0">
    <div class="highlight-bash notranslate">
        <div class="highlight">
            <pre><code class=""><span>touch example-project-directory/public/FIRST_INSTALL</span></code></pre>
        </div>
    </div>
</div>
<div aria-labelledby="tab-owershell" class="sphinx-tabs-panel group-tab" name="owershell" id="panel-owershell" role="tabpanel" tabindex="0">
    <div class="highlight-powershell notranslate">
        <div class="highlight">
            <pre><code class=""><span>echo $null &gt;&gt; public/FIRST_INSTALL</span></code></pre>
        </div>
    </div>
</div>
</div>

    </div>

<!-- content end -->